.gitignore

```diff
@@ -1,6 +1,6 @@
-pganalyze-collector
-pganalyze-collector-helper
-pganalyze-collector-setup
+/pganalyze-collector
+/pganalyze-collector-helper
+/pganalyze-collector-setup
```

Comment:
This was causing the `contrib/helm/pganalyze-collector` dir to be ignored. Adding `/` will enforce that only the root dir one is going to be ignored.